src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -264,31 +274,17 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
      */
     function updateOperators(address[] calldata operators) external onlyWhenNotPaused(PAUSED_UPDATE_OPERATOR) {
         for (uint256 i = 0; i < operators.length; i++) {
-
             address operator = operators[i];
+            // Get operator info and bitmap of quorums operator is registered for
             OperatorInfo storage operatorInfo = _operatorInfo[operator];
             bytes32 operatorId = operatorInfo.operatorId;
-
-            // Only update operators currently registered for at least one quorum
-            if (operatorInfo.status != OperatorStatus.REGISTERED) {
-                continue;
-            }
```

Comment:
seems that by removing this we would now revert if any of the operators in the array is not registered, but perhaps we should confirm on that.
I liked this code since it deals ~fairly elegantly with the race condition where an operator de-registers while someone is trying to perform an update -- it would definitely not be ideal to have a large update (say 5 million gas) revert very late.